os/board/common/README.md

```diff
@@ -0,0 +1,114 @@
+# How to partition a flash into a config
+
+This document provides a way to register a partial block driver in Flash.
+
+## Prepare
+
+To partition Flash, you must first enable ARCH_USE_FLASH. So if new board use FLASH, add 'select ARCH_USE_FLASH' to Kconfig. 
+
+## Contents
+
+> [How to Configure CONFIG](#how-to-configure-config)  
+> [Partition size](#Partition-size)  
+> [Partition type](#partition-type)  
+> [Partition name](#partition-name)  
+
+## How to Configure CONFIG
```

Comment:
```
## How to use

Flash partition is provided by configurations, CONFIG_A, CONFIG_B and CONFIG_C.
CONFIG_A represents size of each partition, B does name, C does interface between flash and upper layer.
All partition information is separated by .....

### CONFIG_A
.....
### CONFIG_B
...
```

Reply:
###CONFIG_A and ###CONFIG_B seem to be a overlap with ##partition size and ##Partition type.